clash-cores/test/unittests.hs

```diff
@@ -18,7 +19,8 @@ import qualified Test.Cores.Xilinx.DcFifo
 
 tests :: TestTree
 tests = testGroup "Unittests"
-  [ Test.Cores.SPI.tests
+  [ Test.Cores.CRC.tests
```

Comment:
Hehe, it looks like the unit tests now take three times as long to run. It's almost at a point where I'd move it to a self-hosted runner rather than a free shared runner provided by GitLab.